bootstrap/pkg/apis/apps/group.go

```diff
@@ -39,7 +39,7 @@ import (
 const (
 	DefaultNamespace = "kubeflow"
 	// TODO: find the latest tag dynamically
-	DefaultVersion     = "master"
+	DefaultVersion     = "v0.5-branch"
```

Comment:
should we use branch name? or v0.5.0 tag?

Reply:
I think "v0.5.0"

Reply:
updated.

Reply:
If we are about to create new release tag v0.5.1, should we point to v0.5.1 instead?
Seems there are several cherry picks on v0.5 branch post v0.5.0.

Reply:
sure, let me know.